events.go

```diff
@@ -110,6 +110,9 @@ func (e *eventRemote) Subscribe(ctx context.Context, filters ...string) (ch <-ch
 				Event:     ev.Event,
 			}:
 			case <-ctx.Done():
+				if cerr := ctx.Err(); cerr != context.Canceled {
```

Comment:
Needs to check the cause: `errors.Cause(cerr) == context.Canceled)`.

Reply:
@stevvooe when the `ctx.Done()` is closed, the `ctx.Err()` will explaining why, canceled or deadline passed. It's no need to use `pkg/errors` or check nil here.